static/script.js

```diff
@@ -364,3 +366,58 @@ function alertIfDuplicateFileName(files) {
     }
     return true
 }
+
+function replaceUsersAndGroups(files) {
+    const regex = /.*(SET_GROUP|SET_OWNER)=['"]?(.*[^'"])['"]?$/gm;
```

Comment:
Is there  a reason why you do not specify `TUNNEL_` here? I don't see any and I would be prefer to be explicit.

Reply:
Sure. I did this because I started off thinking maybe there is something else that uses users and groups, but there is nothing else, right?